spec.yaml

```diff
@@ -299,3 +301,62 @@ paths:
                 "id": "02:cc:da:f9:fb:ce:c3:e5:e1:f6:27:d8:43:48:0d:37:4a:ee:b9:67",
                 "ttl_hours": 8640
               }
+
+  /v4/organizations/:
+    post:
+      operationId: addOrganization
+      tags:
+        - organizations
+      summary: Create an organization
+      description: |
+        This operation allows a user to create an organization.
+      parameters:
+        - name: body
+          in: body
+          required: true
+          description: |
+            An ID for the organization to be created is the only required parameter.
+            This ID can only contain alpha-numeric characters, hyphens, underscores, and dots.
+          schema:
+            $ref: 'definitions.yaml#/definitions/V4AddOrganizationRequest'
+      responses:
+        "201":
+          description: Organization created
+          schema:
+            $ref: "definitions.yaml#/definitions/V4GenericResponse"
+          examples:
+            application/json:
+              {
+                "code": "RESOURCE_CREATED",
+                "message": "A new organization has been created with ID 'OrgID'"
```

Comment:
According to your API PR the message format is this. 

```
"The organization with ID '%s' has been created."
```

See https://github.com/giantswarm/api/pull/386/files#diff-004e9e81a905ac1c76d8c0e8fdb0b4a7R126.

You also miss the period at the end of the sentence.